### ISSUE TITLE
[Suggestion] A way to destroy unnecessary or excess power

### ISSUE THREAD

**Issue by deathaplenty** [BluSunrize/ImmersiveEngineering#2030]
tl;dr I suggest some sort of ground rod or other method of "destroying" unneeded power to prevent surges caused by the buffers on connectors.

I've been trying to power my base with a massive thermo-electric generator array, and it works... mostly. I have enough generators to fully power an excavator. However, I noticed that on occasion, my MV wire connecting to my transformer to step up to HV burns out. At first, I thought it was just buggy, so I'd move the transformer around, or hook up something new, or unhook something else. 

Now, I've realized the issue:

If I stop drawing power from my generators, the buffers on my connectors fill up. If I then use more power than the max MV cable can carry ( for example, powering two excavators), the 98 MV connectors on my generators try to dump all of the power in their buffers, burning the wire. Since I'm trying to achieve a constant flow of power and have no need of power storage, I'd like to see some way to "destroy" unnecessary power in order to prevent this surge problem. I'm imagining a grounding rod that dumps RF that isn't being used by other constructs.

**Comment by mindforger** [comment]
You should never bundle more than 4 cables this is bound to burn

**Comment by deathaplenty** [comment]
I've fixed my own issue by hooking each hv-equivalent thermo array to an hv capacitor via 4 MV connectors, thus defeating the entire purpose of transformers. My suggestion still stands though. Realistically, electrical systems are grounded to provide a return path for electricity. Otherwise, a return wire is necessary.

**Comment by theunkn0wn1** [comment]
Well, ive asked for something similar for over volt protection (a circuit breaker/fuse box) but Blu never got around to it(see #457). So i'd wager that this suggestion won't be a high priority for him either. 

The idea isn't bad, but the problem only exists with poorly designed energy networks to begin with. The simple solution to prevent needing something like these is not to design systems that just cannot work under load to begin with.

**Comment by AEnterprise** [comment]
i could use a way to destroy excess power, mainly for testing new setups and configurations

but as server admin i'm a bit worried that this could be trolling instrument, if someone hooks this up to someone elses base it could completely wreck their power generation (especially if he/she has a generator setup that uses fuel faster then is produced and relies on not having to run all the time due to buffers)

**Comment by TheGuyRunningSouth** [comment]
I think I saw this implemented by a different mod at some point but don't
remember which one off the top of my head. If anybody wants me to try and
find it again in case Blu turns this down let me know, otherwise I'm not
gonna go out of my way to name drop a different mod on here.

**Comment by deathaplenty** [comment]
@AEnterprise You can set a creative capacitor to accept energy for testing your setups. That's how I fix my problems or test new setups.

**Comment by AEnterprise** [comment]
@deathaplenty i know but sometimes i have a build done in survival but i need to test it, for example when tweaking a farm it's usefull to know how fast a constantly running generator would eat through the backlog

**Comment by Pihrra** [comment]
@AEnterprise Extra Utilities has an energy trashcan you could use for that purpose. Personally I prefer to make a few batteries and sink everything into them when testing to avoid just wasting power but I never go for crazy powergen so RF is rather precious to me.

**Comment by MalkContent** [comment]
sounds more like you need to regulate your power draw rather than destroy excess power.
the way i do it is putting a capacitor in front of the subsection whichs power draw i want to control, and the power draw of the subsection is then limited by the connectors on the capacitor(s)
